include/tsid/formulations/inverse-dynamics-formulation-base.hpp

```diff
@@ -72,40 +82,40 @@ namespace tsid
     typedef tasks::TaskContactForce TaskContactForce;
     typedef tasks::TaskActuation TaskActuation;
     typedef tasks::TaskBase TaskBase;
+    typedef measuredForces::MeasuredForceBase MeasuredForceBase;
     typedef contacts::ContactBase ContactBase;
     typedef solvers::HQPData HQPData;
     typedef solvers::HQPOutput HQPOutput;
     typedef robots::RobotWrapper RobotWrapper;
 
-
-    InverseDynamicsFormulationBase(const std::string & name,
-                                   RobotWrapper & robot,
-                                   bool verbose=false);
+    InverseDynamicsFormulationBase(const std::string &name,
+                                   RobotWrapper &robot,
+                                   bool verbose = false);
 
     virtual ~InverseDynamicsFormulationBase() {}
 
-    virtual Data & data() = 0;
+    virtual Data &data() = 0;
 
     virtual unsigned int nVar() const = 0;
     virtual unsigned int nEq() const = 0;
     virtual unsigned int nIn() const = 0;
 
-    virtual bool addMotionTask(TaskMotion & task,
+    virtual bool addMotionTask(TaskMotion &task,
                                double weight,
                                unsigned int priorityLevel,
-                               double transition_duration=0.0) = 0;
+                               double transition_duration = 0.0) = 0;
 
-    virtual bool addForceTask(TaskContactForce & task,
+    virtual bool addForceTask(TaskContactForce &task,
```

Comment:
Please revert this "cosmetic" changes. I won't write it anymore but there are plenty of these.